addons/base_import_module/models/ir_module.py

```diff
@@ -134,6 +134,16 @@ def _import_module(self, module, path, force=False, with_demo=False):
                         convert_sql_import(self.env, fp)
                     elif ext == '.xml':
                         convert_xml_import(self.env, module, fp, idref, mode, noupdate)
+                        if f"{module}/{filename}" in terp.get('cloc_exclude',[]):
+                            fp.seek(0)
+                            root = lxml.etree.fromstring(fp.read())
+                            for element in root.xpath("//templates | //record | //template"):
```

Comment:
```suggestion
                        for element in root.xpath("//templates | //record | //template"):
```

Why do you need to parse the xml, you have all the needed information in id_ref just need to iterate on the key of idref

Reply:
How do I get the model?

Reply:
From the xml_id that has been just created

Reply:
If you are referring to the xml_id in the idref it does not include the model

Reply:
yes but you have the xml_id so you can read in ir.model.data the res_id and model